akka-http-core/src/main/scala/akka/http/impl/engine/server/ServerTerminator.scala

```diff
@@ -77,6 +77,31 @@ private[http] final class MasterServerTerminator(log: LoggingAdapter) extends Se
     }
   }
 
+  /**
+   * Removes a previously registered per-connection terminator.
+   * Terminators must remove themselves like this once their respective connection is closed,
+   * otherwise they would leak and remain in the set indefinitely.
+   *
+   * If called with terminator that is not registered, false is returned but it is not an error.
+   *
+   * @return true if the terminator has been successfully removed.
+   */
+  @tailrec def removeConnection(terminator: ServerTerminator)(implicit ec: ExecutionContext): Boolean = {
```

Comment:
Is this ExecutionContext used?

Reply:
It's not used, right; all other methods here needed one, though true that this one not; removing param